scipy/stats/_morestats.py

```diff
@@ -1613,11 +1613,11 @@ def yeojohnson_llf(lmb, data):
     loglike[~tiny_variance] = (
         -n_samples / 2 * np.log(trans_var[~tiny_variance]))
     loglike[~tiny_variance] += (
-        (lmb - 1) * (np.sign(data) * np.log(np.abs(data) + 1)).sum(axis=0))
+        (lmb - 1) * (np.sign(data) * np.log1p(np.abs(data))).sum(axis=0))
     return loglike
 
 
-def yeojohnson_normmax(x, brack=(-2, 2)):
+def yeojohnson_normmax(x, brack=None):
```

Comment:
We discussed in https://github.com/scipy/scipy/pull/18852#issuecomment-1629863631 and the reply that although this looks like a change, it should not materially change the solution found because the problem is strictly convex.

Reply:
In addition to @mdhaber's comment: if a `brack` is supplied, the implementation uses `optimize.brent` as before.